debezium-testing/pom.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <parent>
+    <groupId>io.debezium</groupId>
+    <artifactId>debezium-parent</artifactId>
+    <version>1.1.0-SNAPSHOT</version>
+    <relativePath>../pom.xml</relativePath>
+  </parent>
+  <modelVersion>4.0.0</modelVersion>
+  <artifactId>debezium-testing</artifactId>
+  <name>Debezium Testing</name>
+  <packaging>pom</packaging>
+  <modules>
+    <module>debezium-testing-testcontainers</module>
```

Comment:
I don't recall, are we going to backport testcontainers too?

Reply:
That was my understanding after talking to @gunnarmorling